src/frontend/i18n/locales/es-ES.json

```diff
@@ -35,6 +35,10 @@
     "description": "Helper text in the enroll button for non logged in users",
     "message": "Log in to enroll"
   },
+  "components.CourseRunEnrollment.courseStartsInDays": {
+    "description": "Helper text for days remaining before course starts",
+    "message": "The course will start in {days} day(s)"
```

Comment:
you can pluralize with ReactIntl. See https://github.com/openfun/richie/blob/1e5279fadb8e01df46f65a355567aa90720493d2/src/frontend/i18n/locales/fr-FR.json#L140